server/lib/images.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns true if image is a valid OC image uploaded in our S3 bucket.
+ * Allow any image on non-production environments
+ */
+export const isValidOCImage = (url: string): boolean => {
```

Comment:
While we're at it, not a fan of names containing `OC`, remove or use `OpenCollective`. Suggested names: `isValidUploadedImage`. Or, maybe it should be more generic and be `isValidUploadedAttachment`

Reply:
Going with `isValidUploadedImage`